### ISSUE TITLE
bare (non-css?) version of page flickers every time page is loaded

### ISSUE THREAD

**Issue by petercl1000** [mediarox/pagespeed#16]
Great module, nicely improved speed! Works out of the box.

But every time a page (any page) is loaded, for half a second a 'bare' version of the page flickers on the screen before that page is shown well. 

This:
![screen](https://cloud.githubusercontent.com/assets/19282261/15140720/b7156f2e-169c-11e6-8f49-8316b4b59465.jpg)

Any suggestions how I could fix that? 

Best regards

Peter

**Comment by tomas213** [comment]
Did u managed to find a sollution for that?

**Comment by petercl1000** [comment]
No, I disabled the css settings. And only enabled the javascript settings.

**Comment by airoptix** [comment]
I also have the same problem with ultimo theme. If someone has the solution, please share with me! 
Thanks! 

Even in mail: david.feher.84@gmail.com

**Comment by NendoTaka** [comment]
This is probably due to the CSS being pushed to the bottom of the page. Since all of the elements are being loaded before the CSS, they will have no styles. Once the styles are loaded, the elements are being updated.

**Comment by sasoriza** [comment]
You need to inline the critical css needed to display the above the fold content in <head> to fix this. This might help: https://jonassebastianohlsson.com/criticalpathcssgenerator/
